### ISSUE TITLE
dedicated decorator for specifying parser for a single mapper

### ISSUE THREAD

**Issue by gamboviol** [klbostee/dumbo#35]
(Used to be "parser attribute on a single mapper gets applied to others in same MultiMapper".)

**Comment by klbostee** [comment]
Yeah, that's actually sort of by design. Dumbo options always apply to the whole program, and attaching them to individual mappers is basically just a convenient way of specifying that those options should be set for each program that uses the mapper.

It should be straightforward to introduce a dedicated parser decorator that doesn't use the options system and directly applies a given parser to a mapper though. Patches or pull requests are welcome.. :)
